### PR TITLE
Improve imputation workflow

### PR DESCRIPTION
**Pull request by pjbradbury**
## Description

_Provide a summary of your changes including motivation and context.
If these changes fix a bug or resolves a feature request, be sure to link to that issue._

## Type of change

_What type of changes does your code introduce? Put an `x` in boxes that apply._

- [ ] `CHANGE` (fix or feature that would cause existing functionality to not work as expected)
- [ ] `FEATURE` (non-breaking change which adds functionality)
- [ ] `BUGFIX` (non-breaking change which fixes an issue)
- [x] `ENHANCEMENT` (non-breaking change which improves existing functionality)
- [ ] `NONE` (if none of the other choices apply. Example, tooling, build system, CI, docs, etc.)

## Checklist:

- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have updated relevant documentation

## Changelog entry

_Please add a one-line changelog entry below. This will be copied to the changelog file during the release process._

```release-note

```

### CODE REVIEW

File: src/main/kotlin/net/maizegenetics/phgv2/pathing/BuildKmerIndex.kt [hunk 1]
```diff
@@ -39,25 +39,27 @@ import kotlin.time.measureTimedValue
...
+        "to <hvcfDir>/kmerIndex.txt") {
 
     private val myLogger = LogManager.getLogger(BuildKmerIndex::class.java)
 
     val dbPath by option(help = "Tile DB URI")
         .required() //Needs to be required now due to the agc archive
 
     val maxHaplotypeProportion by option("-p", "--maxHapProportion", help = "only kmers mapping to less than or " +
-            "equal to maxHapProportion of haplotypes in a reference range will be retained.")
+            "equal to maxHapProportion of haplotypes in a reference range will be retained.Defualt = 0.75")
```
Comment (zrm22) [inline src/main/kotlin/net/maizegenetics/phgv2/pathing/BuildKmerIndex.kt:51]: Typo with Default here.

Comment (zrm22) [reply src/main/kotlin/net/maizegenetics/phgv2/pathing/BuildKmerIndex.kt:51]: When asking for help, Clikt might even provide the default automatically without it needing to be in the description.

**Review by zrm22 — APPROVED** [review]

**Comment by zrm22** [comment]
Also double check your unit tests.  It looks like 2 are failing.

File: src/main/kotlin/net/maizegenetics/phgv2/pathing/BuildKmerIndex.kt (hunk 1 above)
Comment (pjbradbury) [reply src/main/kotlin/net/maizegenetics/phgv2/pathing/BuildKmerIndex.kt:51]: Clickt --help does not provide either default values or information about whether or not a parameter is required.

File: docs/imputation.md [hunk 2]
```diff
@@ -90,11 +106,11 @@ along with the cumulative coverage, which is the proportion of reads mapping to
 
 ### Required Parameters
 
-**path-keyfile**: the path to a keyfile that contains a list of the read mapping files. The keyfile must have two columns
-labeled SampleName and ReadMappingFiles. Any additional columns will be ignored. SampleName must be unique.
-Additionally if an output hvcf for a sample name already exists in the output directory, the sample will be skipped.
-If a ReadMappingFiles value is a comma-separated list of file names, the read mappings will be combined.
-The file names must include the full path to the file.
+**key-file** or **read-file** but not both. **key-file** is the name and path of a keyfile that contains a list of the 
+read mapping files or read files (fastq). The keyfile must have two columns labeled sampleName and filename.
```
Comment (tcasstevens) [inline docs/imputation.md:110]: Above, it says there can optionally be a third column "filename2"?

**Review by lynnjo — APPROVED** [review]

**Review by btmonier — APPROVED** [review]

**Pull request merged**